package.json

```diff
@@ -10,8 +10,10 @@
     "test": "echo \"Test will be implemented.\" && exit 0",
     "start": "./bin/gtop",
     "lint": "prettier --single-quote --trailing-comma es5 --write 'lib/**/*.js'",
-    "lint-check": "prettier --check --single-quote --trailing-comma es5 'lib/**/*.js'"
+    "lint-check": "prettier --check --single-quote --trailing-comma es5 'lib/**/*.js'",
+    "man": "cd man && marked-man --version 'v1.1.2' --manual 'Gtop Help' --section 1 man.md > gtop.1"
```

Comment:
is it possible to not to hardcode the version in here?

Reply:
yes, it is possible. I've pushed a commit that address that.